docs/pages/divider.js

```diff
@@ -1,64 +1,388 @@
+      <Block title="Plants" url="https://i.ibb.co/FY2MKr5/stock6.jpg" />
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use Dividers to separate collections of content or create the appearance of containers."
+            defaultCode={`
+function Example() {
+  return (
+    <Flex width="100%" height="100%">
+      <Box paddingX={4}>
+        <Flex direction="column" gap={2} width={120}>
+        {["Public profile", "Account settings", "Home feed tuner", "Claim", "Permissions", "Notifications"].map((item, idx) => <Text underline={idx === 0}weight="bold" key={idx} size="md" wrap="none">{item}</Text>)}
+        </Flex>
+      </Box>
+      <Divider />
+      <Box marginStart={12}>
+        <Flex direction="column" gap={4}>
+          <Heading accessibilityLevel="none" size="md">Public profile</Heading>
+          <Avatar
+            size="lg"
+            src="https://i.ibb.co/ZfCZrY8/keerthi.jpg"
+            name="Keerthi"
+          />
+          <TextField
+            id="first"
+            onChange={() => {}}
+            label="First name"
+            value="Kate"
+          />
+          <TextField
+            id="last"
+            onChange={() => {}}
+            label="Last name"
+            value="Dommeti"
+          />
+        </Flex>
+      </Box>
+    </Flex>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use Dividers for decoration or to separate grouped content."
+            defaultCode={`
+function Example() {
+  return (
+    <Flex width="80%">
+      <Box paddingX={4}>
+        <Heading accessibilityLevel="none" size="sm">Our mission</Heading>
+      </Box>
+      <Divider />
+      <Box paddingX={4}>
+        <Text size="md">Pinterest's mission is to bring everyone the inspiration to create a life they love.</Text>
+      </Box>
+    </Flex>
+  );
+}
+`}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Accessibility">
+        <MainSection.Subsection description="The Divider is not focusable and is treated as decorative. Screen readers on tab navigation don't announce Dividers but do announce them on left/right quick navigation." />
+      </MainSection>
+      <MainSection name="Localization">
+        <MainSection.Subsection description="If you are aligning the Divider to content, ensure that it switches sides according to the content's start-end alignment." />
+      </MainSection>
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="Orientation"
+          columns={2}
+          description="You can use this component for a visual divider between two elements. Placing it within a [Flex](/flex) layout with a direction of `row` will cause the Divider to become vertical."
```

Comment:
The copy seems a little off…perhaps something like:
"You can use this component as a vertical divider between two elements. Placing it within a Flex layout with a direction of `"row"` will shift Divider to a vertical orientation."

Reply:
Done